profiler/include/profiler/profile_pool3d_fwd_impl.hpp

```diff
@@ -229,8 +240,18 @@ bool profile_pool3d_fwd_impl(PoolFwdInputParams& in_params, PoolFwdKernelParams&
         {
             out_device_buf.FromDevice(out_n_c_do_ho_wo_device.mData.data());
 
-            auto absolute_error_threshold = ck::utils::get_absolute_threshold<ComputeDataType>();
-            auto relative_error_threshold = ck::utils::get_relative_threshold<ComputeDataType>();
+            auto absolute_error_threshold = 1.0;
+            switch(in_params.init_method)
+            {
+            case 0: absolute_error_threshold = static_cast<double>(inDataRangeTensor1); break;
+            case 1: absolute_error_threshold = static_cast<double>(inDataRangeTensor2); break;
+            default: absolute_error_threshold = static_cast<double>(inDataRangeTensor3);
```

Comment:
Float is enough in this casting. After all the input parameter is of type `float`.

Reply:
Changed the input parameter to be double. The are some uses later of this parameter requiring the double type, check_err for example